src/Library/demos/Row/main.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Rows";
+  description: "Create user interface lists with ease";
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+      Adw.PreferencesGroup  {
+
+      title: "Rows can be groupped";
+
+      Adw.ActionRow {
+        title: "Action row can have an icon";
+        icon-name: "info";
+      }
+
+      Adw.ActionRow {
+        title: "Action row can have a prefix child";
+
+        [prefix]
+        CheckButton checkbox {
+          active: true;
+          }
+      }
+
+      Adw.ActionRow {
+        title: "Action row can have a suffix child";
+        subtitle: "Select checkbox above";
+
+        [suffix]
+        Spinner {
+          spinning: bind checkbox.active;
+        }
+      }
+
+      Adw.EntryRow {
+         title: "A row can be an entry";
+      }
+    }
+
+    Adw.PreferencesGroup {
+      title: "Groups can have a suffix";
+
+      [header-suffix]
+      Adw.ButtonContent {
+      label: "Settings";
+      icon-name: "org.gnome.Settings-symbolic";
+    }
+
+    /* TODO */
+    Adw.ComboRow {
+      title: "Choose a license";
+      subtitle: "ComboRow can have a menu";
+      /*
+      model: Adw.EnumListModel model {
+        enum-type: typeof(GtkLicense);
+      };
+
+      expression: bind model.nick;*/
+    }
+
+
+    Adw.ExpanderRow {
+      title: "Rows can be expandable";
+
+      // disable this if you don't want a switch
+      show-enable-switch: true;
+
+      Adw.ActionRow {
+        title: "First row";
+      }
+      Adw.ActionRow {
+        title: "Second row";
+      }
+    }
+
+    Adw.ActionRow {
+      title: "Row can have an action type child";
+
+      [suffix]
+      Button {
+      label: "test";
+      // it looks bad right now, needs some magic gtk class (of which I might not know about) or custom css class
+      styles [""]
+      }
+    }
+  }
+
+    LinkButton {
+      label: "API Reference";
+      uri: "https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.PreferencesGroup.html";
+    }
```

Comment:
maybe use this instead https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/boxed-lists.html